spec/features/form/check_answers_page_sales_logs_spec.rb

```diff
@@ -43,8 +43,10 @@
           visit("/sales-logs/#{completed_sales_log_joint_purchase.id}/#{subsection}/check-answers")
           assert_selector ".x-govuk-summary-card__title", text: "Buyer 1", count: 1
           assert_selector ".x-govuk-summary-card__title", text: "Buyer 2", count: 1
-          assert_selector ".x-govuk-summary-card__title", text: "Person 1", count: 1
+          assert_selector ".x-govuk-summary-card__title", text: "Person 1", count: 0
           assert_selector ".x-govuk-summary-card__title", text: "Person 2", count: 0
+          assert_selector ".x-govuk-summary-card__title", text: "Person 3", count: 1
+          assert_selector ".x-govuk-summary-card__title", text: "Person 4", count: 1
```

Comment:
I would have assumed that Person 6 would be allowed to (I thought for a joint purchase there would be Buyer 1, Buyer 2, Person 3, Person 4, Person 5, Person 6) since you can have up to 4 non-buyers. However testing this locally leads to a test failure. Am I missing something?

Reply:
Added person 5, person 6 fails for me too. But that's cause I don't think we're asking questions about person 6. I've added a field for it but the implementation might be part of 1847(?)

Reply:
The reason I thought 6 might be valid is because in `schema.rb` there are fields for the 6th person, for example these can be seen in the sales log table definition:
- t.integer "age6"
- t.integer "age6_known"
- t.integer "ecstat6"
- t.string "relat6"